Client/Frontend/Browser/ScreenshotHelper.swift

```diff
@@ -0,0 +1,12 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+/**
+ * Handles screenshots for a given browser, including pages with non-webview content.
+ */
+protocol ScreenshotHelper {
+    func takeScreenshot(tab: Browser, aspectRatio: Float?, quality: Float) -> UIImage?
```

Comment:
A couple of reasons:
 1. The behavior for `nil` is to use the aspect ratio of the webView. This "default" changes depending on the screen size and layout.
 2. You can't specify defaults in protocols.

I could make the protocol include two different methods, one where `aspectRatio` is missing and another where it's non-optional if you guys think that's cleaner.